test/lexers/typescript-lexer.test.js

```diff
@@ -0,0 +1,23 @@
+import { assert } from 'chai'
+import { testJavaScriptLexer } from './javascript-lexer.test'
+import { testJsxLexer } from './jsx-lexer.test'
```

Comment:
I'm really not a fan of adding any unecessary logic to tests. Would you mind simply duplicating the tests. I don't think DRY principles should apply to tests, simplicity and straightforwardness are far more important.

Reply:
Sure thing